pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -355,9 +355,9 @@ func ConfigPath(ctx *types.SystemContext) string {
 	return systemRegistriesConfPath
 }
 
-// ConfigDirPath returns the path to the system-wide directory for drop-in
-// registry configuration files.
-func ConfigDirPath(ctx *types.SystemContext) string {
+// UserConfigDirPath returns the user's directory for drop-in
+// registry configuration files, returns "" is not specified
+func UserConfigDirPath(ctx *types.SystemContext) string {
 	if ctx != nil && ctx.SystemRegistriesConfDirPath != "" {
```

Comment:
Shouldn't this still be under ConfigDirPath?

Reply:
I split the original method ConfigDirPath(), ConfigDirPath() still there.

Reply:
No I am talking about 

if ctx != nil && ctx.SystemRegistriesConfDirPath != "" {

Should be in the ConfigDirPath() not in UserConfigDirPath()

Reply:
Sorry, I miss read! Will this ctx.SystemRegistriesConfDirPath comes from user specification in libpod or buildah?

Reply:
Yes.

Reply:
So I think it should be under UserConfigDirPath, like the $HOME/.config/containers

Reply:
In general, it should be possible for the caller to disable using the system directory. But I don’t (yet?) think we need separate overrides for the system and per-user one: if the user specifies a specific directory, we can use only that configuration.